src/components/common/_material-overrides.scss

```diff
@@ -91,3 +91,7 @@
     @include elevation.elevation(9);
   }
 }
+
+.material-icons {
+  font-family: 'Material Icons Extended';
```

Comment:
Is `'Material Icons Extended'` a superset? If so, can / shall we remove the Material Icons CSS from `public/index.html`?

Reply:
it's weird, the font family has to be `Material Icons Extended`, but both imports must be present, otherwise it stops showing regular icons.